examples/ex1.c

```diff
@@ -0,0 +1,68 @@
+
+static char help[] = "Trains binary SVM classification model using precomputed Gramian (linear kernel) matrix.\n\
+                      Provided Gramian matrix is computed and related to no-bias SVM formulation (svm_binary_mod 2).\n";
+
+#include <permonsvm.h>
+
+int main(int argc,char **argv)
+{
+  SVM            svm;
+
+  char           training_file[PETSC_MAX_PATH_LEN] = "data/heart_scale.bin";
+  char           kernel_file[PETSC_MAX_PATH_LEN]   = "data/heart_scale.kernel.bin";
+  char           test_file[PETSC_MAX_PATH_LEN]     = "data/heart_scale.t.bin";
+  PetscBool      test_file_set = PETSC_FALSE;
+
+  PetscViewer    viewer;
+  PetscErrorCode ierr;
+
+  ierr = PermonInitialize(&argc,&argv,(char *)0,help); if (ierr) return ierr;
+
+  ierr = PetscOptionsGetString(NULL,NULL,"-f_training",training_file,sizeof(training_file),NULL);CHKERRQ(ierr);
+  ierr = PetscOptionsGetString(NULL,NULL,"-f_kernel",kernel_file,sizeof(test_file),&test_file_set);CHKERRQ(ierr);
```

Comment:
```suggestion
  ierr = PetscOptionsGetString(NULL,NULL,"-f_kernel",kernel_file,sizeof(kernel_file),NULL);CHKERRQ(ierr);
```
otherwise setting kernel without setting test_file will load default test_file and run tests with it

Reply:
Accepted.